db/post_migrate/20230119091939_drop_orphaned_reviewable_flagged_posts.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DropOrphanedReviewableFlaggedPosts < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec(<<~SQL)
+      DELETE FROM reviewables
+      WHERE reviewables.type = 'ReviewableFlaggedPost'
```

Comment:
Only targeting flagged posts here since they’re the ones being problematic when the `AutoQueueHandler` job is called.